### PR TITLE
Added configuration options for ssh_config

### PR DESCRIPTION
**Pull request by bogdanr**
Also made a small change to reflect the default sshd_config

### CODE REVIEW

File: openssh/files/sshd_config [hunk 1]
```diff
@@ -69,7 +68,7 @@
 
 # Authentication:
 {{ option_default_uncommented('LoginGraceTime', 120) }}
-{{ option_default_uncommented('PermitRootLogin', 'no') }}
+{{ option_default_uncommented('PermitRootLogin', 'yes') }}
```
Comment (gravyboat) [inline openssh/files/sshd_config]: What was the reason for this change? Allowing someone to log in as root is a big red flag.

Comment (bogdanr) [inline openssh/files/sshd_config]: This is the default setting in vanilla ssh and in pretty much most major distros. It should be up to the users to move from the default setting.

**Comment by alexproca** [comment]
:+1:

File: openssh/files/sshd_config (hunk 1 above)
Comment (puneetk) [inline openssh/files/sshd_config]: Please update the readme clarifying that people need to explicitly turn off root login, looks ok to merge.

Comment (bogdanr) [inline openssh/files/sshd_config]: Done

**Pull request merged**